facilitator/src/test_utils.rs

```diff
@@ -0,0 +1,78 @@
+use ring::signature::{
```

Comment:
nit: these values are also used to provide default arguments in `src/main/facilitator.rs` so `test_utils.rs` isn't _quite_ right. `default_keys.rs`?

Reply:
Well, my point in the commit message is that they *shouldn't* be defaults there. We shouldn't have things be set up such that things are accidentally encrypted using a compromised key if you forget to specify the flag.

Reply:
Yes, sorry, I really should have read your commits first. I think you make a good point about safety here, and we should work to make it as hard as possible to use this tool wrong. I just did command line flags for everything because as I was writing this, the intended audience was engineers driving it from their shells, but as you work through the execution manager, we'll have to figure out how we want to provide parameters to this program (environment variables? config files mounted into the container? params passed through the `args` array in a k8s job template?) and that strikes me as a good time to decide how to make this safe.